### ISSUE TITLE
'Show touches in recordings' doesn't work with GIF recordings

### ISSUE THREAD

**Issue by medcalc** [AvdLee/RocketSimApp#201]
**Describe the bug**
Touches don't show up in GIF recordings.

**To Reproduce**
1. Check 'Show touches in recordings' in Preferences...
2. Click on 'Start GIF Recording'
3. Tap on simulator while recording, touches are displayed
4. End recording
5. View final result, touches are not shown on the produced GIF file.

**Expected behavior**
Touches should appear, as they do when using MP4 recordings

**Desktop (please complete the following information):**
 - OS: Ventura 13.0 beta 3 (yes sorry it's a beta)
 - Version: RocketSim 8.3.0 (111)

**Comment by medcalc** [comment]
https://digitalbunker.dev/automatically-create-gifs-from-the-ios-simulator/

Now I understand, this feature depends on code you didn't write (Apple code), so that's why this is certainly a "won't fix".

**Comment by medcalc** [comment]
Yes, I detailed the reason I prefer gifs in the first issue I posted ( https://github.com/AvdLee/RocketSimApp/issues/202 ). Mainly animated Gifs are much easier to integrate in websites that MP4, especially in the middle of a slideshow of jpegs. Not having to change the markup for movies is much easier.

I wouldn't mind so much that GIFs are second-class citizen (no device chrome, no touches) if it was stated more explicitely upfront.
